File: Programmers/Level1/Python/행렬의_덧셈.py
def solution(arr1, arr2):
    answer = []
    for i, j in zip(arr1, arr2):
        tmp = []
        for idx in range(len(i)):
            tmp.append(i[idx] + j[idx])

        answer.append(tmp)
        
    return answer

print(solution([[1,2],[2,3]], [[3,4],[5,6]]))

File: Programmers/Level1/Swift/행렬의_덧셈.swift
func solution(_ arr1:[[Int]], _ arr2:[[Int]]) -> [[Int]] {
    var answer = arr1
    for i in 0..<arr1.count{
        for j in 0..<arr1[i].count{
            answer[i][j] = arr1[i][j] + arr2[i][j]
        }
    }
    
    return answer
}